src/js/pouch-express.js

```diff
@@ -0,0 +1,295 @@
+/* eslint-env node */
+/*
+
+    Provide a CouchDB-like API using `PouchDB` and `express-pouchdb`.  This in only useful in association with an
+    existing `gpii.express` instance.  See the documentation for details:
+
+    https://github.com/GPII/gpii-pouchdb/blob/master/docs/pouch-express.md
+
+ */
+"use strict";
+
+/*
+
+    TODO:  Work through the express-pouchdb setup and code with Antranig, as we see many warnings like:
+
+    `Warning: a promise was created in a handler but was not returned from it`
+
+    The workaround for now is to set the `BLUEBIRD_WARNINGS` environment variable to `0`.
+
+ */
+process.env.BLUEBIRD_WARNINGS = 0;
+
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.pouch.express");
+
+var os             = require("os");
+var fs             = require("fs");
+var memdown        = require("memdown");
+
+var expressPouchdb = require("express-pouchdb");
+
+var PouchDB        = require("pouchdb");
+
+var path           = require("path"); // eslint-disable-line no-unused-vars
```

Comment:
Thanks.  Given that I had to tiptoe around the ESLint "unused variable" warning, it should be easy enough to remember when this is relevant in the future.